app/views/handlers/individual_response.py

```diff
@@ -182,30 +205,199 @@ class IndividualResponseHowHandler(IndividualResponseHandler):
         },
     }
 
-    def __init__(self, schema, questionnaire_store, language, form_data, list_item_id):
+    def __init__(
+        self,
+        schema,
+        questionnaire_store,
+        language,
+        request_args,
+        form_data,
+        list_item_id,
+    ):
         super().__init__(
             self.block_definition,
             schema,
             questionnaire_store,
             language,
+            request_args,
             form_data,
             list_item_id,
         )
 
     def handle_get(self):
-        previous_location_url = url_for(
-            "individual_response.request_individual_response",
-            list_item_id=self._list_item_id,
-        )
+        if self._request_args.get("journey") == "change":
```

Comment:
Given this request arg isn't used on _every_ journey through the IR scenarios, only on the change scenario, then it should probably be called something else. `action` might make more sense in this case, such as `action=change`

Reply:
The intention is this param will be used for other journeys. For future stories we will need to know whether they came from the hub or from the individual section.